src/cargo/core/resolver/features.rs

```diff
@@ -319,8 +319,32 @@ impl ResolvedFeatures {
         pkg_id: PackageId,
         features_for: FeaturesFor,
     ) -> Vec<InternedString> {
-        self.activated_features_int(pkg_id, features_for)
```

Comment:
What about reusing and calling the unverified variant in this function. If none then panic?

Reply:
Good idea!